components/automate-cli/cmd/chef-automate/bootstrap.go

```diff
@@ -0,0 +1,101 @@
+package main
+
+import (
+	"bufio"
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+
+	api "github.com/chef/automate/api/interservice/deployment"
+	"github.com/chef/automate/components/automate-cli/pkg/status"
+	"github.com/chef/automate/components/automate-deployment/pkg/client"
+	"github.com/spf13/cobra"
+)
+
+func newBootstrapBundleCmd() *cobra.Command {
+	var bootstrapCmd = &cobra.Command{
+		Use:    "bootstrap COMMAND",
+		Hidden: true,
+	}
+
+	var bundleCmd = &cobra.Command{
+		Use:    "bundle COMMAND",
+		Hidden: true,
+	}
+
+	var createCmd = &cobra.Command{
+		Use: "create <current working dir>/bootstrap_bundle.tar",
+		Annotations: map[string]string{
+			NoCheckVersionAnnotation: NoCheckVersionAnnotation,
+			NoRequireRootAnnotation:  NoRequireRootAnnotation,
+		},
+		RunE:   runBootstrapBundleCreate,
+		Hidden: true,
+	}
+
+	bootstrapCmd.AddCommand(bundleCmd)
+	bundleCmd.AddCommand(createCmd)
+
+	return bootstrapCmd
+}
+
+func runBootstrapBundleCreate(cmd *cobra.Command, args []string) error {
+	connection, err := client.Connection(client.DefaultClientTimeout)
+	if err != nil {
+		fmt.Println("Connection failed")
```

Comment:
do we need to do this? I thought the stuff we wrap the CLI with handles printing out the error